lowell-dashboard/app/models.py

```diff
@@ -217,4 +243,30 @@ class Classes(Model):
     year = Column(Integer)
     course_type = Column(String(64))
     a_g_requirement = Column(String(64))
-'''
+    _students_ids = Column(String)
+
+    '''
+    A getter function for the student id property
+    Args:
+        None
+    Returns:
+        list: a list of the students ids that are in the class
+    Use case:
+    print(Classes.students_ids)
+    ['421', '319']
+    '''
+    @property
+    def students_ids(self):
+        return [float(x) for x in self._students_ids.split(';')]
+    '''
+    A setter for the student id property
+    Args:
+        str: user id
+    Returns:
+        void: nothing
+    Use case:
+    Classes.students_ids = 421
+    '''
+    @students_ids.setter
+    def students_ids(self, value):
+        self._students_ids += ';%s' % value
```

Comment:
also to confirm do you need to get the db object and commit to the db.

Reply:
Good question. The answer is yes. After you change the value of the object you must use the db.commit() function to save changes.